package.json

```diff
@@ -118,6 +118,7 @@
     "ts:check": "tsc --noEmit && tsc -p tsconfig.examples.json --noEmit",
     "start": "tsx --tsconfig tsconfig.examples.json",
     "start:bee": "yarn start -- examples/agents/bee.ts",
+    "start:telemetry": "BEE_FRAMEWORK_INSTRUMENTATION_ENABLED=true node --enable-source-maps --experimental-loader=@opentelemetry/instrumentation/hook.mjs --import ./examples/helpers/telemetry.js",
```

Comment:
When I run `yarn start:telemetry examples/llms/instrumentation.js`

I get 

```
(node:70374) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("%40opentelemetry/instrumentation/hook.mjs", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
(node:70374) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("%40opentelemetry/instrumentation/hook.mjs", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
INF  ℹ️  08:57:34 (app): LLM 🤖 (txt) : Hello! How are you today? Is there something I can help you with or would you like to chat?
INF  ℹ️  08:57:39 (app): Process exiting after OpenTelemetry flush.
```

Reply:
You can easily change the command to use `tsx` instead of `node`


`BEE_FRAMEWORK_INSTRUMENTATION_ENABLED=true tsx --import ./examples/helpers/telemetry.js`

and update `examples/helpers/telemetry.js` by adding this at the very first line 

```
import "@opentelemetry/instrumentation/hook.mjs";
```

It seems that integrating with `TypeScript` is very straightforward. Ideally, you should rename JS files to TS.